pytest.ini

```
@@ -0,0 +1,6 @@
+[pytest]
```

This should be used together with, so it fails in case there is a typo in the markers

```suggestion
[pytest]
addopts = --strict-markers
```